persist-cli-tests/src/test/resources/test-src/output/tool_test_generate_13/modules/clients/multipleassociations_client.bal

```diff
@@ -24,85 +24,75 @@ public client class MultipleAssociationsClient {
-        stream<anydata, error?>|error result = self.persistClient.runReadQuery(MultipleAssociations, (), include);
-        if result is error {
-            return new stream<MultipleAssociations, error?>(new MultipleAssociationsStream((), result));
+    remote function read(MultipleAssociationsRelations[] include = []) returns stream<MultipleAssociations, persist:Error?> {
+        stream<anydata, sql:Error?>|persist:Error result = self.persistClient.runReadQuery(MultipleAssociations, include);
+        if result is persist:Error {
+            return new stream<MultipleAssociations, persist:Error?>(new MultipleAssociationsStream((), result));
         } else {
-            return new stream<MultipleAssociations, error?>(new MultipleAssociationsStream(result));
+            return new stream<MultipleAssociations, persist:Error?>(new MultipleAssociationsStream(result));
         }
     }
 
-    remote function execute(sql:ParameterizedQuery filterClause) returns stream<MultipleAssociations, error?> {
-        stream<anydata, error?>|error result = self.persistClient.runExecuteQuery(filterClause, MultipleAssociations);
-        if result is error {
-            return new stream<MultipleAssociations, error?>(new MultipleAssociationsStream((), result));
+    remote function execute(sql:ParameterizedQuery filterClause) returns stream<MultipleAssociations, persist:Error?> {
+        stream<anydata, sql:Error?>|persist:Error result = self.persistClient.runExecuteQuery(filterClause, MultipleAssociations);
+        if result is persist:Error {
+            return new stream<MultipleAssociations, persist:Error?>(new MultipleAssociationsStream((), result));
         } else {
-            return new stream<MultipleAssociations, error?>(new MultipleAssociationsStream(result));
+            return new stream<MultipleAssociations, persist:Error?>(new MultipleAssociationsStream(result));
         }
     }
 
-    remote function update(MultipleAssociations value) returns error? {
-        map<anydata> filter = {"id": value.id};
-        _ = check self.persistClient.runUpdateQuery(value, filter);
+    remote function update(record {} value) returns persist:Error? {
+        _ = check self.persistClient.runUpdateQuery(value);
         if value["profile"] is record {} {
-            record {} profileEntity = <record {}>value["profile"];
+            Profile profileEntity = <Profile>value["profile"];
             ProfileClient profileClient = check new ProfileClient();
-            stream<MultipleAssociations, error?> multipleAssociationsStream = self->read([ProfileEntity]);
-            check from MultipleAssociations p in multipleAssociationsStream
-                do {
-                    if p.profile is Profile {
-                        check profileClient->update(<Profile>profileEntity);
-                    }
-                };
+            check profileClient->update(profileEntity);
         }
         if value["user"] is record {} {
```

Comment:
```suggestion
        if value.user is record {} {
```

Reply:
same reason as the above, prevents from replacing the syntax